lightning/src/ln/msgs.rs

```diff
@@ -724,6 +724,17 @@ enum EncodingType {
 	Uncompressed = 0x00,
 }
 
+/// An onion message to be sent or received from a peer
+#[derive(Clone, Debug, PartialEq)]
+pub struct OnionMessage {
+	/// The blinding point used in the shared secret that is used to decrypt the onion message's
+	/// `encrypted_data` field.
+	pub blinding_point: PublicKey,
```

Comment:
Does this need to be a `Result<PublicKey...`? ie if we receive an onion message on the wire we shouldn't disconnect our peer just because the point is invalid (I mean our peers shouldnt do it, but still).